src/components/my-view3.js

```diff
@@ -45,12 +46,30 @@ class MyView3 extends connect(store)(PageViewElement) {
           border-color: var(--app-primary-color);
           color: var(--app-primary-color);
         }
+        .shroud svg path#first {
+            fill: var(--app-primary-color);
+        }
+        .shroud {
+          width: 64px;
+          height: 64px;
+        }
+        .shroud svg {
+          width: 64px;
+          height: 64px;
+        }
+        .circle.small {
+          margin-top: -72px;
+          width: 28px;
+          height: 28px;
+          font-size: 16px;
+          font-weight: bold;
+          line-height: 30px;
+        }
       </style>
 
       <section>
         <h2>Redux example: shopping cart</h2>
-        <div class="circle">${_quantity}</div>
-
+        <div class="shroud">${addToCartIcon}<div class="circle small">${_quantity}</div></div>
```

Comment:
Do you mind showing a screenshot of this? I'm not sure what htis means or how it looks like :)